src/components/ControlInput.vue

```diff
@@ -486,9 +487,12 @@ export default {
             if (opts.commands) {
                 let commandList = [];
                 autocompleteCommands.forEach((command) => {
+                    let desc = command.description.startsWith('command_') ?
+                        TextFormatting.t(command.description) :
+                        command.description;
```

Comment:
I've added this so the autocompleteCommands file can be overridden with static strings

Reply:
This should be a comment in the code itself